pkg/middleware/oapi_validate_test.go

```diff
@@ -218,6 +218,46 @@ func TestOapiRequestValidatorWithOptions(t *testing.T) {
 		rec := doPost(t, r, "http://example.com/protected_resource_401", body)
 		assert.Equal(t, http.StatusUnauthorized, rec.Code)
 		assert.False(t, called, "Handler should not have been called")
+		assert.Contains(t, rec.Result().Header["Content-Type"][0], ErrRespContentTypePlain)
+		called = false
+	}
+
+	// Confirm explicit text/plain content-type is returned
+	{
+		options.ErrRespContentType = ErrRespContentTypePlain
+		body := struct {
+			Name int `json:"name"`
+		}{
+			Name: 7,
+		}
```

Comment:
Since this specific body is used multiple times, maybe we can pull it out of the scopes and just reuse it